recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>I would recommand to add the used gammapy version in the header of this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>